src/components/DataCollector/validateUserEventOptions.js

```diff
@@ -22,7 +33,20 @@ export default ({ options, logger }) => {
   const eventOptionsValidator = objectOf({
     type: string(),
     xdm: objectOf({
-      eventType: string()
+      eventType: string(),
+      identityMap: mapOfValues(
+        arrayOf(
+          objectOf({
+            id: string().required(),
```

Comment:
ID is not listed as required in the XDM schema

Reply:
ok, will relax the validation a little bit